hydraw/hydraw.cabal

```diff
@@ -73,8 +73,9 @@ executable hydraw
     -Wunused-packages -threaded -rtsopts
 
   build-depends:
-    , base            >=4.7 && <5
+    , base               >=4.7 && <5
     , http-types
+    , hydra-cardano-api
```

Comment:
Why is this additional depends required?

Reply:
for the `NetworkId` argument to `paintPixel` for example.